lib/Models/CswCatalogGroup.js

```diff
@@ -0,0 +1,788 @@
+"use strict";
```

Comment:
Should we keep this deleted given it's useless in mobx?

Reply:
(The file, not this line itself)

Reply:
Oh whoops. Did I revive a deleted file? I didn't mean to.

Reply:
Seems so!